.github/workflows/nightlies.yml

```diff
@@ -70,7 +70,7 @@ jobs:
       - name: Upload wheels to Anaconda Cloud as nightlies
         run: |
           anaconda --token ${{ secrets.ANACONDA_ORG_UPLOAD_TOKEN }} upload \
-            --user scipy-wheels-nightly \
+            --user scientific-python-nightly-wheels \
```

Comment:
@jarrodmillman I realize that this PR was made the same day as https://github.com/scientific-python/upload-nightly-action was released, but given that it exists, why not just implement it in this PR with switching this step out with

```yaml
      - name: Upload wheels to Anaconda Cloud as nightlies
        uses: scientific-python/upload-nightly-action@0.1.0
        with:
          artifacts_path: dist
          anaconda_nightly_upload_token: ${{ secrets.ANACONDA_ORG_UPLOAD_TOKEN }}
```

? If that's too much of a hassle given that this PR is already approved as is and works, then I can also PR this later in follow up with also fixing the docs.

Reply:
Maybe best to leave that to a separate PR since this was already approved. But if it would be better to do here, I am more than happy to make the change. @tacaswell Thoughts?